### DISCUSSION TITLE
Add a Github Sponsor Option

### DISCUSSION THREAD

**Discussion by jstrong013** [PSAppDeployToolkit/PSAppDeployToolkit#710] [Ideas]
First and foremost, I appreciate all of the [contributors ](https://github.com/PSAppDeployToolkit/PSAppDeployToolkit/graphs/contributors) on this tool! I'm aware of the [donation page](https://psappdeploytoolkit.com/donate/). 

With that said, I think it would be cool to see a [GitHub Sponsor](https://docs.github.com/en/sponsors/getting-started-with-github-sponsors/about-github-sponsors) option in addition to the webpage. 

Thoughts on the addition?

**Comment by mjr4077au** [comment]
This hasn't been answered, but the reason it likely hasn't is that sponsorship from the community puts undue pressure on developers to implement the features a sponsor demands, which may or may not be feasible, either within a timely manner or ever if it's against the spirit of the project. It's better that it remains sponsorless to ensure the project maintains a neutral disposition and agenda.
